pgpool2-4.6.yaml

```diff
@@ -63,6 +68,59 @@ subpackages:
       runtime:
         - ${{package.name}}
 
+  - name: ${{package.name}}-compat
+    description: "Compatibility package for Upstream pgpool image"
+    dependencies:
+      provides:
+        - pgpool2-compat=${{package.full-version}}
+      runtime:
+        - postgresql
+        - postgresql-client
+        - postgresql-dev
```

Comment:
Concerning that the Postgres version will just change under people's feet, but upstream appears to be okay with that as they're doing the same.

Reply:
Agreed, I've moved `postgres` (17) to main package's runtime deps as per your suggestion.